scripts/hail_batch/variant_selection/hgdp_1kg_tob_wgs_variant_selection.py

```diff
@@ -51,10 +51,18 @@ def query(output):  # pylint: disable=too-many-locals
     hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.filter_rows(
         (hl.len(hgdp1kg_tobwgs_joined.alleles) == 2)
         & (hgdp1kg_tobwgs_joined.locus.in_autosome())
-        & (hgdp1kg_tobwgs_joined.variant_qc.AF[1] > 0.001)
+        & (hgdp1kg_tobwgs_joined.variant_qc.AF[1] > 0.01)
         & (hgdp1kg_tobwgs_joined.variant_qc.call_rate > 0.99)
         & (hgdp1kg_tobwgs_joined.IB.f_stat > -0.25)
     )
+
+    hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.cache()
+    nrows = hgdp1kg_tobwgs_joined.count_rows()
+    print(nrows)
```

Comment:
It's good to give context for whatever you print out, to make it easier to make sense of it later:

```suggestion
    print(f'hgdp1kg_tobwgs_joined.count_rows() = {nrows}')
```